trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0, 10]
```

Comment:
- Please, do ensure the name has a maximum of 10 characters. If it's longer it should trim the word, as of now, yours is 9 characters. please, fix this.

Reply:
@zieeco About this, I think the current code is working correctly as it reduces the name by a maximum of 10 characters. It reduces 'Maximilianus' with 12 characters to 'Maximilian' with 10 characters.😀 

![image](https://user-images.githubusercontent.com/97407840/186277150-3b0fc836-8ac5-474c-9a64-b7e745a5a07a.png)